test/main_entry.res

```diff
@@ -0,0 +1,96 @@
+/* let load = Main_counter.main (Web.Document.getElementById "content") () */
```

Comment:
What's happening with this file?

Reply:
this was commented out in the bucklescript-testing repo should I remove this file ?

Reply:
What does it do?

Reply:
It starts by loading some test files using this function

example:

`let load = Main_counter.main (Web.Document.getElementById "content")`

Because the code was commented out it is still in Ocaml. Tried to uncomment it in order to covert it to rescript and see what it does exactly didn't work. 

Error when converting test/main_entry2.ml
File "", line 33, characters 1-4:
Error: Syntax error

This error is not only in this line it is everywhere.